src/libostree/ostree-deployment.c

```diff
@@ -113,6 +162,14 @@ ostree_deployment_set_bootconfig (OstreeDeployment *self, OstreeBootconfigParser
     self->bootconfig = g_object_ref (bootconfig);
 }
 
+/**
+ * ostree_deployment_set_origin:
+ * @self: Deployment
+ * @origin: (nullable): Set the origin for this deployment
```

Comment:
This duplicates the description. Maybe just

```suggestion
 * @origin: (nullable): Origin object
```

?